build/ci/windows/package.bat

```diff
@@ -47,7 +47,7 @@ IF %BUILD_WIN_PORTABLE% == ON (
 
 :: Setup package type
 IF %BUILD_WIN_PORTABLE% == ON    ( SET PACKAGE_TYPE="portable") ELSE (
-IF %BUILD_MODE% == devel_build   ( SET PACKAGE_TYPE="7z") ELSE (
+IF %BUILD_MODE% == devel_build   ( SET PACKAGE_TYPE="msi") ELSE (
```

Comment:
temporary for the test will be back

Reply:
For testing in my repo, where available secrets
https://github.com/igorkorsukov/MuseScore/pull/81